<?xml version="1.0"?>

<launch>
  <node name="action_recognition" pkg="action_recognition" type="action_recognition_node.py" output="screen">
    <rosparam command="load" file="$(find action_recognition)/launch/action_recognition_params.yaml"/>
  </node>
</launch>
